core/src/main/java/nl/eernie/jmoribus/parser/GherkinsListener.java

```diff
@@ -2,26 +2,12 @@
 
 import nl.eernie.jmoribus.GherkinsBaseListener;
 import nl.eernie.jmoribus.GherkinsParser;
-import nl.eernie.jmoribus.model.Feature;
-import nl.eernie.jmoribus.model.Line;
-import nl.eernie.jmoribus.model.Prologue;
-import nl.eernie.jmoribus.model.Scenario;
-import nl.eernie.jmoribus.model.Step;
-import nl.eernie.jmoribus.model.StepContainer;
-import nl.eernie.jmoribus.model.StepType;
-import nl.eernie.jmoribus.model.Story;
-import nl.eernie.jmoribus.model.Table;
+import nl.eernie.jmoribus.model.*;
```

Comment:
Well my Intelij decided that I wanted a other formatter, and I can't seem to find the old formatter :(